bootstrap-salt.sh

```diff
@@ -1324,6 +1324,18 @@ __check_dpkg_architecture() {
         "amd64")
             error_msg=""
             ;;
+        "arm64")
+            if [ "$_CUSTOM_REPO_URL" != "null" ]; then
+                warn_msg="Support for arm64 is experimental, make sure the custom repository used has the expected structure and contents."
+            else
+                # Saltstack official repository does not yet have arm64 metadata,
+                # use amd64 repositories on arm64, since all pkgs are arch-independent
+                __REPO_ARCH="amd64"
```

Comment:
Probably adding `amd64` as a "foreign" architecture is missing here.
You need to check if it is not listed in `dpkg --print-foreign-architectures` and then add it with the
`dpkg --add-architecture` command.

Reply:
Adding amd64 as a foreign architecture is not needed. I actually think that would be a bad idea to apply here, as we don't want any x86 binaries on arm64, we just tell APT to fetch the repository metadata from the x86-specific URL (which includes 'binary-amd64'), but we only care about the arch-independent packages in that repo.